### PR TITLE
Update the readme

### PR DESCRIPTION
**Pull request by guy-patrick**
remove the :green_square: and replaced it with green squares text

closes #ISSUE-NO

#### What does this PR do?
remove the :green_square: and replaced it with green squares text

### CODE REVIEW

File: discord-readme.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ COMMUNITY DISCUSSION:
 
 -> #organisation: our community's GitHub org
 
--> #good-first-issues: get started with open source and get your :green_square: - opportunities are provided for new contributors - discussions not to happen here as this is just for people to share links, please discuss in #first-timers
+-> #good-first-issues: get started with open source and get your green squares opportunities are provided for new contributors - discussions not to happen here as this is just for people to share links, please discuss in #first-timers
```
Comment (Vyvy-vi) [inline discord-readme.md:62]: Is this change really needed?

Comment (guy-patrick) [reply discord-readme.md:62]: maybe not, I was trying to do some update on the readme according to what Eddie wrote on discord. Maybe I didn't get what it was all about.

Comment (schmelto) [reply discord-readme.md:62]: I would leave it with the emoji maybe adding an explanation like :green_square:  (green square) opertunity.

**Review by adityaraute — COMMENTED** [review]
In the very next line the emoji 🟩 is used. Shows inconsistency. 

I think I agree with @Vyvy-vi here. This change is not really improving the file.

File: discord-readme.md (hunk 1 above)
Comment (kaiwalyakoparkar) [reply discord-readme.md:62]: I agree with @Vyvy-vi ! @schmelto That is also not necessary but if more descriptive is need then adding it is a good option

Comment (eddiejaoude) [reply discord-readme.md:62]: I can't remember what I wrote on Discord 🤦‍♂️ . I think the combination of the icon and the text is more descriptive as suggested by the community 👍

**Comment by kaiwalyakoparkar** [comment]
Any activity on this PR?

File: discord-readme.md (hunk 1 above)
Comment (guy-patrick) [reply discord-readme.md:62]: you're right.
